packages/library/tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
```

Comment:
Do we need tailwind config in library?

Reply:
yeap, its gonna be used by the `tailwindcss` CLI to transpile the tailwind classes, and most importantly not include the default tailwind classes